primitives/npos-elections/Cargo.toml

```diff
@@ -18,6 +18,7 @@ serde = { version = "1.0.101", optional = true, features = ["derive"] }
 sp-std = { version = "2.0.0", default-features = false, path = "../std" }
 sp-npos-elections-compact = { version = "2.0.0", path = "./compact" }
 sp-arithmetic = { version = "2.0.0", default-features = false, path = "../arithmetic" }
+sp-core = { version = "2.0.0", default-features = false, path = "../core" }
```

Comment:
I only had to add this because I realized I was deriving `Debug` (rather than `RuntimeDebug`) which is frowned upon. But I also don't like adding the core just for that, might re-consider.

Reply:
I think it is fine to import it